webook/arrangement/dto/event.py

```diff
@@ -0,0 +1,27 @@
+import time
+from dataclasses import dataclass, field
+from datetime import datetime
+from typing import List
+
+
+@dataclass
+class EventDTO:
+    title: str
+    start: datetime
+    end: datetime
+
+    title_en: str = ""
+
+    rooms: list = field(default_factory=list)
+    people: List[int] = field(default_factory=list)
+    display_layouts: list = field(default_factory=list)
```

Comment:
Correct, making a default factory is the correct way to handle having a mutable default value with dataclass. 
another way to solve it is to have `Optional[list] = None` and create a new list in the `__post_init__` method that is unique to dataclasses.

Reply:
It has not been tested as well as I would have wanted, but the current bugs that had appeared in production were more pressing, and any bugs present are now at the very least not as much of a roadblock. For instance repeating events did not work properly, and dialogs did not close properly (header remained after close.)

I did a test of the most common functionalities yesterday after I merged with dev and fixed what I found, and I will continue testing today, and will most likely have another PR by EOD that we can push in tomorrow morning.